res/database/Default/database_scripts.xml

```diff
@@ -470,76 +470,582 @@
+							<en>ambitious and determined, sets high goals and achieves them|very enthusiastic, with contagious passion|very organized with impressively precise planning skills|enthusiastic, finds joy in the little things|intuitive, always has the right instincts|ambitious, strives to change the world for the better|creative, ideas bubble up inexhaustibly|diplomatic, mediates conflicts and brings people together|curious, absorbs everything and constantly discovers new things|authentic, always remains true and does not pretend</en>
+						</character3>
+					</variables>
 				</scripttemplate>
 				<scripttemplate index="1" product="2" licence_type="2" guid="scripttemplate-random-ron-averagedayseries01_02">
 					<title>
-						<de>%SUBJECT1A% im %SUBJECT1B%</de>
-						<en>%SUBJECT1A% in %SUBJECT1B%</en>
+						<de>%trouble% im ${location_${k}}</de>
+						<en>%trouble% in ${location_${k}}</en> <!-- TODO: apply capitalization function when available -->
 					</title>
+					<description>
+						<de>%descriptions%</de>
+					</description>
 					<episodes min="1" max="3" />
 					<variables>
-						<subject1a>
-							<de>Ärger|Panik|Pechsträhne|Fette Beute</de>
-							<en>Trouble|Panic|Streak of bad luck|Big booty</en>
-						</subject1a>
-						<subject1b>
-							<de>Paradies|Gasthaus|Hinterhof|Polizeirevier|Supermarkt</de>
-							<en>paradise|the roadhouse|the backyard|the police station|the supermarket</en>
-						</subject1b>
+						<descriptions>
+							<de>%description_0%|%description_1%</de>
+						</descriptions>
+						<description_0>
+							<de>%gang% %rioting% %bikers% %appears%${whythere_${k}}%causingtrouble%. Die ${who_${i}} %solvetheconflict% %intheirownway%.</de>
+							<en>%gang% %rioting% %bikers% %appears%${whythere_${k}}%causingtrouble%. The ${who_${i}} %solvetheconflict% %intheirownway%.</en>
+						</description_0>
+						<description_1>
+							<de>%park% in %town% ist %asocialhotspot%. Die ${who_${i}} treffen auf ${who_${j}} aus %othertown% und %discuss% %fiercely% über %politics%.</de>
+							<en>In %town%, %park% is %asocialhotspot%. The ${who_${i}} encounter ${who_${j}} from %othertown% and %fiercely% %discuss% %politics%.</en>
+						</description_1>
+						<trouble>
+							<de>Ärger|Panik|Pechsträhne|Fette Beute|Chaos|Durcheinander|Desaster|Aufruhr|Krise|Aufregung|Verwüstung|Unruhe|Tumult|Eklat|Zoff|Krawall</de>
+							<en>Trouble|Panic|Streak of bad luck|Big booty|Chaos|Mayhem|Disaster|Turmoil|Crisis|Commotion|Havoc|Unrest|Tumult|Uproar|Down and out|Riot</en>
+						</trouble>
+						<j>
+							<de>0|1|2|3|4|5|6|7|8|9|a|b|c|d|e|f|g|h</de> <!-- must be same as i -->
```

Comment:
is "i" defined at all;
The editor validation for variables is not optimal, yet!!

Reply:
yes, it's defined above, globally for the series

Reply:
OK. So "must be same as i" reads as " same options as i" rather than "same chosen value as i"...